examples/drivers/windows/otLwf/radio.c

```diff
@@ -759,6 +759,13 @@ ThreadError otPlatRadioEnergyScan(_In_ otInstance *otCtx, uint8_t aScanChannel,
     return NT_SUCCESS(status) ? kThreadError_None : kThreadError_Failed;
 }
 
+void otPlatRadioSetDefaultTxPower(otInstance *aInstance, int8_t aPower)
+{
+    // TODO: Create a proper implementation for this driver.
+    (void)aInstance;
```

Comment:
This will require a corresponding Spinel Property and NCP code to handle it, device side. As I just finished getting all Radio APIs matching Spinel/NCP code, I personally would prefer that new Radio APIs and there corresponding Spinel/NCP code be added at the same time.